server/cast/numeric.go

```diff
@@ -84,7 +84,7 @@ func numericExplicit() {
 			if d.LessThan(pgtypes.NumericValueMinInt64) || d.GreaterThan(pgtypes.NumericValueMaxInt64) {
 				return nil, fmt.Errorf("bigint out of range")
 			}
-			return int64(d.IntPart()), nil
+			return d.IntPart(), nil
```

Comment:
Although redundant, I had it the other way _just in case_ it was ever changed at some point in the future. I'm very confident that it won't, but it'll be a **hard** bug to track down if it ever did, since somewhere else will expect an `int64` and error/panic, so it would have to be traced back to this particular line, so this just guards us against that.